main.py

```diff
@@ -2,6 +2,33 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George",
+                         last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu",
+                         last_name="Reaves").update(first_name="Keanu",
+                                                    last_name="Reeves")
+
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(name="Scarlett").delete()
+
+
+    return Actor.objects.filter(last_name="Smith").order_by()
```

Comment:
The `order_by()` method is called without any arguments. This will not change the order of the queryset. If you intend to order the results, you should specify a field name, such as `order_by('first_name')` or `order_by('last_name')`.